src/components/Footer.js

```diff
@@ -1,12 +1,19 @@
 import "./Footer.css";
+import { NavLink } from "react-router-dom";
 
 export default function Footer() {
   return (
-    <footer className="App-footer">
-      <ul className="App-footer__navigation">
-        <li>MyCakes</li>
-        <li>Plus</li>
-        <li>Favs</li>
+    <footer className="Footer">
+      <ul className="Footer__navigation">
+        <NavLink to="/">
+          <li>MyCakes</li>
+        </NavLink>
+        <NavLink to="/add">
+          <li>Plus</li>
+        </NavLink>
+        <NavLink to="/favs">
+          <li>Favs</li>
+        </NavLink>
```

Comment:
At the moment you are not really writing a list , since you are writing: 

<ul> 
  <a> 
    <li> some text </li>
  </a>
</ul>

So instead of the ul you could just use a "nav" element  and instead of the "li" elements you would just write <NavLink to="wherever"> Favs </Navlink>

Reply:
Thanks, Jacob! I was planning to implement that at a later stage. But maybe it is a good idea to implement it directly!